src/WebDriverProxy.ts

```diff
@@ -0,0 +1,297 @@
+import * as express from 'express';
```

Comment:
Let's put this in `lib/`, with the idea that modules a user might directly import can be in in the base directory, and supporting code can go in `lib/`.

Reply:
This has been moved.